### ISSUE TITLE
Package.json bad formatted

### ISSUE THREAD

**Issue by IgorDePaula** [jedireza/generator-hapi-style#17]
I created a hapi project by this generator, but the package.json created is bad formatted, i put the keywords question in blank, and the generator created bad code.

**Comment by jedireza** [comment]
Thanks for opening an issue. Would you be willing to share more specific details about how the `package.json` file was formatted incorrectly and how the generated code is bad? Also which generator is this for (App, API, Web, Plugin)?

**Comment by IgorDePaula** [comment]
Sorry, the app generator, the others generators I not tested yet.

I answered all app generator questions, on final, I would install the
modules, but I received of bad fomatted json error on package.json. I
answered to keyword question with enter key, answer blank. THe package.json
has keyword with value: [#&45#&904].

I removed this value and try install the modules again. And I got it.

2015-07-27 13:53 GMT-03:00 Reza Akhavan <notifications@github.com>:
